cloudigrade/api/tests/tasks/maintenance/test_migrate_account_numbers_to_org_ids.py

```diff
@@ -0,0 +1,153 @@
+"""Collection of tests for tasks.maintenance.delete_cloud_accounts_not_in_sources."""
+import http
+import json
+import os
+from unittest import mock
+from unittest.mock import patch
+
+import faker
+from django.test import TestCase
+
+from api.models import User
+from api.tasks import maintenance
+from util.tests import helper as util_helper
+
+_faker = faker.Faker()
+
+
+class MigrateAccountNumbersToOrgIdsTest(TestCase):
+    """Migrate Account Numbers to Org Ids test case."""
+
+    tenant_translator_scheme = "http"
+    tenant_translator_host = "localhost"
+    tenant_translator_port = "8892"
+    tenant_translator_url = (
+        f"{tenant_translator_scheme}://"
+        f"{tenant_translator_host}:{tenant_translator_port}/"
+        f"internal/orgIds"
+    )
+
+    @mock.patch.dict(os.environ, {"TENANT_TRANSLATOR_SCHEME": tenant_translator_scheme})
+    @mock.patch.dict(os.environ, {"TENANT_TRANSLATOR_HOST": tenant_translator_host})
+    @mock.patch.dict(os.environ, {"TENANT_TRANSLATOR_PORT": tenant_translator_port})
```

Comment:
If we change these to load from `settings` (see other comments), we'll need to updated these decorators to look like:
```
@override_settings(TENANT_TRANSLATOR_SCHEME=tenant_translator_scheme)
```

…and same with the other test functions here.